tests/unit/test_initialize.py

```diff
@@ -75,6 +81,22 @@ def test__initialize__deprecated_positional(clean_client_state: None) -> None:
         assert _client_state.jwt_token is not None
 
 
+def test__initialize__updated_environ(clean_client_state: None) -> None:
+    base_url = "https://internal-api.kolena.io"
+    mock_response = Response()
+    mock_response.status_code = 200
+    mock_response._content = str.encode(json.dumps(dataclasses.asdict(FIXED_TOKEN_RESPONSE)))
+    with patch.dict("os.environ", {API_BASE_URL_ENV_VAR: base_url}, clear=True):
+        with patch("requests.put", return_value=mock_response) as patched:
+            kolena.initialize("random entity", "def")
+            client_state = get_client_state()
+            assert client_state.api_token == "def"
+            assert client_state.base_url == API_BASE_URL
+
+        patched.assert_called_once()
+        assert patched.call_args.args[0] == get_endpoint_with_baseurl(API_BASE_URL, "token/login")
```

Comment:
This test would fail before fix as:

```
            patched.assert_called_once()
>           assert patched.call_args.args[0] == get_endpoint_with_baseurl(API_BASE_URL, "token/login")
E           AssertionError: assert 'https://internal-api.kolena.io/v1/token/login' == 'https://api.kolena.io/v1/token/login'
E             - https://api.kolena.io/v1/token/login
E             + https://internal-api.kolena.io/v1/token/login
E             ?         +++++++++

test_initialize.py:98: AssertionError
```